src/main/java/features/language.feature

```diff
@@ -1,12 +1,123 @@
+      | Русский | Эволюция курсов, Конвертер, Курсы валют, Банки, Валютные кассы, Валюты                 | Про нас, Получить курс, Контакты           | Твой проводник в мире валют               | График эволюции валюты Евро в Banca Nationala относительно Молдавский лей      | Относительно: | Банк:  |
+      | English | Rates evolution, Convertor, Exchange rates, Banks, Exchange offices, Currencies        | About us, Get rates, Contacts              | Your daily source of exchange information | Graph of Euro rates evolution in Banca Nationala relative to Moldovan Leu      | Relative to:  | Bank:  |
+      | Română  | Grafic evolutii, Convertor valutar, Curs valutar, Bănci, Case de schimb valutar, Valute| Despre noi, Preluare curs valutar, Contact | Sursa ta zilnică de informații valutare   | Graficul evolutiei valutei Euro la Banca Nationala raportat la Leu moldovenesc | Raportat la:  | Banca: |
+
+  @langConverter
+  Scenario Outline: Language changed at tab 'Convertor'  (#REQ-104)
+    Given main page is opened
+    When user changes page to 'Convertor'
+    Then page was changed to 'Convertor'
+    And  user changes language to '<lang>'
+    And menu translates to chosen language and consists of '<links>'
+    And tab name changes to '<tab>'
+    And header name changed to '<header>'
+    And slogan changed to '<slogan>'
+    And operation options were translated to chosen language '<operations>'
+    And name of convertor changes to chosen language '<convertors>'
+    Examples:
+      | lang    | links                                                                                   | header                                     |tab                         | slogan                                    | operations                                                                                        | convertors         |
+      | Русский | Эволюция курсов, Конвертер, Курсы валют, Банки, Валютные кассы, Валюты                  | Про нас, Получить курс, Контакты           | Curs.md - Конвертер        | Твой проводник в мире валют               | операции за наличные, официальный курс, операции с банковскими картами, операции по перечислению  | Конвертер          |
+      | English | Rates evolution, Convertor, Exchange rates, Banks, Exchange offices, Currencies         | About us, Get rates, Contacts              | Curs.md - Convertor        | Your daily source of exchange information | cash operations, official exchange rate, operations with cards, operations on accounts            | Convertor          |
+      | Română  | Grafic evolutii, Convertor valutar, Curs valutar, Bănci, Case de schimb valutar, Valute | Despre noi, Preluare curs valutar, Contact | Curs.md - Convertor valutar| Sursa ta zilnică de informații valutare   | operatii in numerar, curs oficial, operatii cu carduri bancare, operatii prin virament            | Convertor valutar  |
```

Comment:
Too much unnecessary validations. 
Once you marked this one as REQ-104 it should have only related validations
REQ-104 should only check the language of headers (h1 tags) and tab names on every page